salt/core/ntp.sls

```diff
@@ -1,4 +1,24 @@
-# Disable pre-installed systemd-timesyncd as we prefer NTP.
+{% if grains['osrelease']|float >= 20.04 %}
+# systemd-timesyncd should be installed by default unless the NTP set up code below has ran against server.
+systemd-timesyncd:
+  pkg.installed:
+    - name: systemd-timesyncd
+  service.running:
+    - enable: True
+
+/etc/systemd/timesyncd.conf:
+  file.replace:
+    - pattern: "^#?NTP=.*"
```

Comment:
This works for the default file (which I found at the bottom of https://dannyda.com/2021/04/27/how-to-change-ntp-server-on-ubuntu-20-04-1-lts-simply-default-contents-of-etc-systemd-timesyncd-conf-on-ubuntu-20-04-1-lts/) but `NTP=` can be stated multiple times, in which case I think this won't have the desired behavior.

Reply:
I have updated this to put our NTP servers in the `timesyncd.conf.d` directory and can confirm it is taking effect using the following command `timedatectl show-timesync` (this command only works on Ubuntu 20.04).